src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+}
```

Comment:
All styles above redundant, try to delete this container element and you will see that nothing will change